### PR TITLE
Belt resprite

### PR DESCRIPTION
**Pull request by AmoryBlaine**
Resprites all the base belts, removes the syndicate belt. Die red belt, die.

### CODE REVIEW

**Comment by AmoryBlaine** [comment]
!review

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -461,6 +450,16 @@
...
  	item_state = "fannypack_yellow"
 
+/obj/item/storage/belt/champion
+	name = "championship belt"
+	desc = "Proves to the world that you are the strongest!"
+	icon_state = "championbelt"
+	item_state = "champion"
+	storage_slots = 1
+	can_hold = list(
+		"/obj/item/clothing/mask/luchador"
```
Comment (NonQueueingMatt) [inline code/game/objects/items/weapons/storage/belt.dm]: This should not be a string.

Comment (AmoryBlaine) [reply code/game/objects/items/weapons/storage/belt.dm]: Sorry, wrong button. How do I fix it?

Comment (NonQueueingMatt) [reply code/game/objects/items/weapons/storage/belt.dm]: Remove the `"`s.

**Pull request closed without merging**